src/common/Color.h

```diff
@@ -398,27 +398,27 @@ class Token
 	{}
 
 	/*
-	 * Pointer to the first character of this token in the input sequence
+	 * RawToken defines the extent of the token as it was written in the original string. It
```

Comment:
Perhaps you can document when to use each version of the API? RawToken vs NormalizedToken vs PlainText

Reply:
Added more doc